modules/multi-architecture-modify-machine-set-gcp.adoc

```diff
@@ -123,27 +126,35 @@ Use the `project` and `name` parameters from the output to create the path to im
 $ projects/<project>/global/images/<image_name>
 ----
 <4> Optional: Specify custom metadata in the form of a `key:value` pair. For example use cases, see the GCP documentation for link:https://cloud.google.com/compute/docs/metadata/setting-custom-metadata[setting custom metadata].
-<5> Specify an ARM64 supported machine type. For more information, refer to _Tested instance types for GCP on 64-bit ARM infrastructures_ in "Additional resources".
+<5> Specify a machine type that aligns with the CPU architecture of the chosen AMI. For more information, see "Tested instance types for GCP on 64-bit ARM infrastructures".
```

Comment:
AMI is AWS-specific terminology.

```suggestion
<5> Specify a machine type that aligns with the CPU architecture of the chosen OS Image. For more information, see "Tested instance types for GCP on 64-bit ARM infrastructures".
```

ref: https://cloud.google.com/compute/docs/images

Reply:
We have only 1 instance of "AMI" in the GCP doc.
I guess this was directly suggested by @jeffdyoung 
Will let him also take a look.

Reply:
yeah, I missed this. GCP shouldn't have "AMI"

Reply:
Updated as per the suggestion.